doc/Customization-guide/How-to-contribute/Git-flow/index.md

```diff
@@ -54,16 +76,22 @@ git push
 
 GitHub will automatically update your pull request.
 
-### Tests
+### Testing
 
-All pull requests are automatically checked with Travis and Jenkins. If any of the tests fails, fix it or describe why that can't be done.
+All pull requests are automatically tested using [Travis](https://travis-ci.org/) and [Jenkins](https://jenkins.io/).
+In case some tests fail, fix the issues or describe why the fix cannot be done.
 
 ### Review
 
-The pull request must have at least two LGTM (Looks Good To Me) from the members of the team responsible for a changed component and should have no unresolved review questions.
+Every pull request is reviewed by the assigned team members.
+When reviewers have comments regarding a PR, they post the comments on the PR page for the author to incorporate them in the PR.
+When reviewers have no (more) comments regarding the PR, they posts an 'LGTM' comment on the PR page.
+A PR can be merged when it receives the 'LGTM' comment from at least two assigned reviewers.
```

Comment:
We're currently transferring to [GitHub pull request reviews](https://help.github.com/articles/about-pull-request-reviews/), which allow explicit approval. So the rule is having at least two approved reviews and no requested code changes (rejected reviews).